eis_qgis_plugin/processing/eis_processing_algorithm.py

```diff
@@ -110,6 +111,14 @@ def prepare_arguments(self, parameters: Dict, context):
                 [args.append(os.path.normpath(layer.source())) for layer in layers]
                 continue
 
+            # TODO check if works
+            elif isinstance(param, QgsProcessingParameterPoint):
+                center_coords = self.parameterAsPoint(parameters, name, context)
+                latitude = center_coords.y()
+                longitude = center_coords.x()
+                args.append(latitude)
+                args.append(longitude)
```

Comment:
Should not be named "center_coords", could be any coordinates. This might be changed later, but for now I would do it like this:
```suggestion
                coords = self.parameterAsPoint(parameters, name, context)
                arg = (coords.x(), coords.y())
```